frontend/app/src/components/inputs/ExchangeInput.vue

```diff
@@ -0,0 +1,46 @@
+<script lang="ts" setup>
+const props = defineProps<{
+  value: string | null;
+}>();
+
+const emit = defineEmits<{
+  (e: 'input', value: string): void;
+}>();
+
+const vModel = useSimpleVModel(props, emit);
+const { exchangesWithKey }
+  = storeToRefs(useLocationStore());
+
+const rootAttrs = useAttrs();
+</script>
+
+<template>
+  <VAutocomplete
```

Comment:
Didn't we already had a component to select exchanges? what are the differences? is that a normal select?

Reply:
Scratch it I see that this is the component it was not just extracted before